File: service-facturation/micro-service/Models/Patient.cs
using MongoDB.Bson.Serialization.Attributes;

namespace micro_service.Models
{
    public class Patient
    {
        [BsonElement("prenom")]
        public string prenom { get; set; } = string.Empty;

        [BsonElement("nom")]
        public string nom { get; set; } = string.Empty;

        [BsonElement("email")]
        public string email { get; set; } = string.Empty;

        [BsonElement("numSecu")]
        public string numSecu { get; set; } = string.Empty;

        [BsonElement("numTel")]
        public string numTel { get; set; } = string.Empty;

        [BsonElement("dateNaissance")]
        public string dateNaissance { get; set; } = string.Empty;

        [BsonElement("genre")]
        public string genre { get; set; } = string.Empty;

        [BsonElement("idMedecinTraitant")]
        public int idMedecinTraitant { get; set; }

        [BsonElement("antecedents")]
        public string antecedents { get; set; } = string.Empty;
    }
}

File: service-gestion-rdv-patients-personnel/src/main/java/com/example/modele/Patient.java
package com.example.modele;

import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "patient")
public class Patient extends Utilisateur{
    private String numSecu;
    private String numTel;
    private String dateNaissance;
    private String genre;
    private int idMedecinTraitant;
    private String antecedents;
    public Patient(String prenom, String nom, String email, String numSecu, String numTel, String dateNaissance, String genre) {
        super(prenom, nom, email);
        this.numSecu = numSecu;
        this.numTel = numTel;
        this.dateNaissance = dateNaissance;
        this.genre = genre;
    }
    public String getNumSecu() {
        return numSecu;
    }
    public void setNumSecu(String numSecu) {
        this.numSecu = numSecu;
    }
    public String getNumTel() {
        return numTel;
    }
    public void setNumTel(String numTel) {
        this.numTel = numTel;
    }
    public String getDateNaissance() {
        return dateNaissance;
    }
    public void setDateNaissance(String dateNaissance) {
        this.dateNaissance = dateNaissance;
    }
    public String getGenre() {
        return genre;
    }
    public void setGenre(String genre) {
        this.genre = genre;
    }
    public int getIdMedecinTraitant() {
        return idMedecinTraitant;
    }
    public void setIdMedecinTraitant(int idMedecinTraitant) {
        this.idMedecinTraitant = idMedecinTraitant;
    }
    public String getAntecedents() {
        return antecedents;
    }
    public void setAntecedents(String antecedents) {
        this.antecedents = antecedents;
    }

    @Override
    public String toString() {
        return "Patient{" +
                "nom='" + this.getNom() + '\'' +
                ", prenom='" + this.getPrenom() + '\'' +
                ", email='" + this.getEmail()+ '\'' +
                ", numSecu='" + numSecu + '\'' +
                ", numTel='" + numTel + '\'' +
                ", dateNaissance='" + dateNaissance + '\'' +
                ", genre='" + genre + '\'' +
                ", idMedecinTraitant=" + idMedecinTraitant +
                ", antecedents='" + antecedents + '\'' +
                '}';
    }
}